app/data/census_household.json

```diff
@@ -1429,12 +1429,17 @@
                                                 {
                                                     "label": "Yes, an address outside the UK (please specify)",
                                                     "value": "Other",
-                                                    "other": {
-                                                        "label": "Please enter country"
-                                                    }
+                                                    "child_answer_id": "another-address-answer-other"
```

Comment:
You're right, potentially manual errors could occur from this. But this two way relationship is however used in multiple places, such as checking where error messages should be mapped and which answer ids are valid for a form. I can't see a cleaner route to doing this.